.github/workflows/deepseek-cr.yaml

```
@@ -1,23 +1,95 @@
 
 permissions:
   pull-requests: write
 
 jobs:
   code-review:
     runs-on: ubuntu-latest
-    name: Code Review
+    name: Code Review2
     steps:
       - name: Checkout Code
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)